src/global/tracking/tracking.cc

```diff
@@ -76,43 +76,32 @@ void InitPlugin(JApplication *app) {
             "CentralTrackerSourceLinker"
         },
         {
-            "CentralCKFSeededTrajectories"
+            "CentralCKFSeededTrajectories",
+            "CentralCKFSeededActsTrajectories",
         },
         app
     ));
 
     app->Add(new JChainFactoryGeneratorT<TrackProjector_factory>(
-            {"CentralCKFTrajectories"}, "CentralTrackSegments"));
+            {"CentralCKFActsTrajectories"}, "CentralTrackSegments"));
 
     app->Add(new JChainFactoryGeneratorT<IterativeVertexFinder_factory>(
-            {"CentralCKFTrajectories"}, "CentralTrackVertices"));
-
-    app->Add(new JChainMultifactoryGeneratorT<TrackingResult_factory>(
-            "CentralTrackingParticles",                       // Tag name for multifactory
-            {"CentralCKFTrajectories"},                       // ActsExamples::Trajectories
-            {"outputTrackParameters"},                        // edm4eic::TrackParameters
-            app));
+            {"CentralCKFActsTrajectories"}, "CentralTrackVertices"));
 
     app->Add(new JChainMultifactoryGeneratorT<ParticlesWithTruthPID_factory>(
             "ChargedParticlesWithAssociations",                // Tag name for multifactory
             {"MCParticles",                                    // edm4hep::MCParticle
-            "outputTrackParameters"},                          // edm4eic::TrackParameters
+            "CentralCKFActsTrajectories"},                          // edm4eic::TrackParameters
```

Comment:
I don't understand this part. `CentralCKFActsTrajectories ` is a vector of `ActsExamples::Trajectories*` (ACTS multitrajectory). So, I assume we need to rewrite to convert it `edm4eic::TrackParameters` type before passing it to this factory.

Reply:
?

Reply:
Just an incorrect comment. It needs Acts trajectories.

Reply:
Ok, I see what you mean. Let me revisit and clarify. There is indeed a part missing. The track parameters are linked in the trajectories (both acts and podio) and the PID matching should use the podio trajectories as a start.

Reply:
Resolved in https://github.com/eic/EICrecon/pull/860/commits/0406b6523d7f1609894b71611b965b1d6027d6a5.